modules/ROOT/pages/_partials/security/proc_performing-device-checks.adoc

```diff
@@ -476,7 +476,7 @@ Executing multiple checks asynchronously is not supported at the moment for this
 
 === Adding Custom Device Checks
 
-Besides the xref:security/index.adoc#ref_terminology_Device%20Security_ref_terminology[Provided Device Checks] it's also possible to make use of your own custom checks. Follow the next steps depending on your platform to implement them:
+Besides the xref:glossary.adoc#ref_terminology_Device Security_ref_terminology[Provided Device Checks] it's also possible to make use of your own custom checks. Follow the next steps depending on your platform to implement them:
```

Comment:
I'm thinking this should either be link: or that there's a glossary section included in security master assembly and the xref would be internal to security master assembly

Reply:
I see that "Performing Device Trust Checks in your Device" has snippets required to implement the chosen features, while glossary holds info about those features. Idk if this can be written better, what was the motivation behind separating these two?

Reply:
i think we used to include glossary in the "masters"

Reply:
is this ok then?

Reply:
should be link: instead of xref, I think
reason: xref works in antora, but will not work with asciidoctor
but feel free to test

Reply:
aha ok will do. I thought that there is a problem with the text. will test, if ok ill merge

Reply:
built glossary and device-security with asciidoctor, it seems to work